workos/mfa.py

```diff
@@ -127,6 +126,11 @@ def verify_factor(
         Returns: Dict containing the  challenge factor details.
         """
 
+        warn(
+            "'verify_factor' is deprecated. Please use 'verify_challenge' instead.",
+            DeprecationWarning,
+        )
+
         params = {
```

Comment:
Given the differences between `verify_factor` and `verify_challenge` are...

* One has a `warn` since it is deprecated
* The other uses the new `auth/challenges/{challenge_id}/verify` endpoint, but is otherwise backwards compatible

Can we define `veryify_factor` in terms of `verify_challenge` until it is finally removed?

```py
    def verify_factor(
        self,
        authentication_challenge_id=None,
        code=None,
    ):
        """
        Verifies the one time password provided by the end-user.

        Kwargs:
            authentication_challenge_id (str) - The ID of the authentication challenge that provided the user the verification code.
            code (str) - The verification code sent to and provided by the end user.

        Returns: Dict containing the  challenge factor details.
        """

        warn(
            "'verify_factor' is deprecated. Please use 'verify_challenge' instead.",
            DeprecationWarning,
        )

        return self.verify_challenge(
           authentication_challenge_id=autentication_challenge_id, code=code
        )
```

Reply:
That's a good point.

I updated `verify_factor` to be implemented in terms of `verify_challenge`.